cli/azd/resources/keys/public-key-83e3efa9.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
can you help me understand why we need this public keys embedded ?  I saw the `verifySignature()` loading this public keys.  Are these keys coming from the extension source you created for testing?

Reply:
The public key can be used to validate the integrity of the registry file through the root level `signature` property.  When the registry is updated the signature is generated using the private key using the content of the registry file.  This validation is only performed when the `signature` field is present.  

If an extension source does not provide a signature - then it is not validated.

We still some work to do to figure out the process.